### DISCUSSION TITLE
Doom binary for TX16s

### DISCUSSION THREAD

**Discussion by DangerD1024** [EdgeTX/edgetx#4526] [General]
Where i can  download compiled binary of Doom for TX16S

**Comment by pfeerick** [comment]
It's on the EdgeTX Discord in the [doom-dev-competition channel](https://discord.com/channels/839849772864503828/990143132706422815/1004016692743127070) , or you can grab the copy of the file I just uploaded => https://filetransfer.io/data-package/y2LYQyxa#link
